utils/image_prep_utils.go

```diff
@@ -157,7 +162,11 @@ func (p CloudPrepper) getConfig() (ConfigSchema, error) {
 		return ConfigSchema{}, err
 	}
 
-	img, err := ref.NewImage(nil)
+	cwd, _ := os.Getwd()
+	ctx := &types.SystemContext{
+		DockerCertPath: cwd,
```

Comment:
Only comment here is that it will fail if your PWD has any certs in it (which I found out the hard way.) I might change my PR to set this to the output of `ioutil.TempDir()` just to be safe.